### PR TITLE
fix: calcuating difference average, av sync and remove unnecessary fields for subscriber stats sample 

### PR DESCRIPTION
**Pull request by eswarclynn**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export class SubscribeStatsAnalytics extends BaseStatsAnalytics {
 
   // eslint-disable-next-line complexity
   private calculateAvSyncForStat(trackStats: HMSTrackStats, hmsStats: HMSWebrtcStats) {
-    if (!trackStats.peerID || !(trackStats.kind === 'video')) {
+    if (!trackStats.peerID || !trackStats.estimatedPlayoutTimestamp || !(trackStats.kind === 'video')) {
```
Comment (raviteja83) [inline packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts:95]: trackStats.kind != 'video'

File: packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts [hunk 2]
```diff
@@ -106,7 +106,10 @@ export class SubscribeStatsAnalytics extends BaseStatsAnalytics {
     if (!audioStats) {
       return MAX_SAFE_INTEGER;
     }
-    return audioStats.timestamp - trackStats.timestamp;
+    if (!audioStats.estimatedPlayoutTimestamp) {
```
Comment (raviteja83) [inline packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts:109]: can we add a code comment

Comment (eswarclynn) [reply packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts:109]: regarding the `estimatedPlayoutTimestamp` property or MAX_SAFE_INTEGER?

Comment (raviteja83) [reply packages/hms-video-store/src/analytics/stats/SubscribeStatsAnalytics.ts:109]: when will this be undefined or 0?

**Pull request merged**